CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Features
 
 * [#114](https://github.com/dotnetinstaller/dotnetinstaller/issues/114) - Added support for operating system product type filters - [@icnocop](https://github.com/icnocop).
 * [#153](https://github.com/dotnetinstaller/dotnetinstaller/issues/153) - Added support for sub-directory installed check comparison using semantic versioning - [@icnocop](https://github.com/icnocop).
+* Building and distributing 64-bit version of CabLib.dll for reference by 64-bit processes - [@icnocop](https://github.com/icnocop).

```

Comment:
Update GitHub issue here?

Reply:
This PR doesn't really fix issue 110.
It's somewhat related, but the real motivation behind these changes are different.
Would you like me to reference issue 110 anyway?

Thank you.

Reply:
You should reference this PR, 156! This way we connect the CHANGELOG line to the PR.

Reply:
Done. Thank you.